### ISSUE TITLE
Travis hangs

### ISSUE THREAD

**Issue by grebe** [ucberkeley-ee290c/fa18-ble#2]
Did you look at the travis output? Any thoughts as to what is going on?

**Comment by jerryduan07** [comment]
Yes I did - It did not hang, it stopped because the output lines exceeded
the limit. The raw file shows that it finished sbt test eventually

**Comment by grebe** [comment]
I don't believe that is accurate. I see this output:

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
